### PR TITLE
Restore UPS extra options

### PR DESCRIPTION
**Pull request by themylogin**
@sonicaj did you drop these on purpose?

### CODE REVIEW

**Comment by sonicaj** [comment]
@themylogin thank you for catching the sql queries. The changes are good. However the `mge-shut` change is not required. Please have a look at ticket 7522. We do not have to explicitly check that anymore.

**Comment by themylogin** [comment]
@sonicaj where did you get that information from?

There are other drivers that are like "xxx or yyy" I wonder what to do with them

**Comment by themylogin** [comment]
@sonicaj I've fixed it right here, please review this again

**Comment by themylogin** [comment]
@sonicaj where?

File: src/middlewared/middlewared/etc_files/local/nut/ups.conf [hunk 1]
```diff
@@ -1,8 +1,13 @@
 <%
 	ups_config = middleware.call_sync('ups.config')
-	driver = ups_config['driver'].strip()
+
+	driver = ups_config['driver'].split('$')[0]
+	driver = driver.split('(')[0]  # "blazer_usb (USB ID 0665:5161)"
+	driver = driver.split(' or ')[0]  # "blazer_ser or blazer_usb"
```
Comment (sonicaj) [inline src/middlewared/middlewared/etc_files/local/nut/ups.conf]: @themylogin here after split

Comment (themylogin) [reply src/middlewared/middlewared/etc_files/local/nut/ups.conf]: `ups_config['driver']` will never contain string with `\n`, why?

Comment (sonicaj) [reply src/middlewared/middlewared/etc_files/local/nut/ups.conf]: `pp(list(map(lambda i:i.split('$')[0].split('(')[0].split(' or ')[0].replace(' ', '\n'), k)))` where k is list of keys from existing ups driver choices. These are one of the few with a space at the end
```
'blazer_usb\n',
 'blazer_usb\n',
 'richcomm_usb\n',
 'blazer_usb\n',
```

Comment (sonicaj) [reply src/middlewared/middlewared/etc_files/local/nut/ups.conf]: `pp` is pprint

**Pull request merged**